docs/src/main/sphinx/release/release-451.md

```diff
@@ -0,0 +1,31 @@
+# Release 451 (25 June 2024)
+
+## General
+
+* Improve performance of parsing JSON arrays and extracting values from them. ({issue}`22379`)
+* Fix error when reading empty files with the native S3 file system. ({issue}`22469`)
+* Fix rare error where query execution could hang when fault-tolerant execution
```

Comment:
Is that worth a release notes entry?

Reply:
Yes, it will help anyone running into an issue like that to know there's some related fix in this release.